lib/builder-webpack4/src/preview/entries.test.ts

```diff
@@ -47,7 +47,7 @@ describe('sortEntries', () => {
       'addons/knobs/dist/preset/addDecorator.js-generated-other-entry.js',
       'addons/a11y/dist/preset/addDecorator.js-generated-other-entry.js',
       'addons/queryparams/dist/preset/addDecorator.js-generated-other-entry.js',
-      'examples/official-storybook/generated-stories-entry.js',
```

Comment:
Doesn't make sense to be changing webpack4 builder... the bug occurs in webpack5, AFAICS based on the title of the PR.

Also webpack4 support will be dropped in 7.0 anyway.